examples/package-lock.json

```diff
@@ -18,6 +18,7 @@
     "../packages/opentelemetry-node": {
       "name": "@elastic/opentelemetry-node",
       "version": "0.3.0",
+      "dev": true,
```

Comment:
Hrm. I'm not sure this should be dev=true.

Reply:
It may slipped in when I was testing locally. Thanks for catching it.